public/styles/_releases.scss

```
@@ -117,7 +133,7 @@
 @media (min-width: 700px) {
   .r-resp-header,
   .r-resp-header-toggle {
```

```suggestion
  .btn-link.r-resp-header-toggle {
```

Can remove `!important` afterwards